spec/v1.0-draft/digest_set.md

```diff
@@ -18,23 +18,22 @@ metadata object.
 ## Fields
 
 A DigestSet is represented as a _JSON object_ mapping algorithm name to
-lowercase hex-encoded value. Usually there is just a single key/value pair,
-but multiple entries MAY be used for algorithm agility.
-
-> **TODO**: Add language for supporting base64-encoded digest values (re: goModuleH1).
+a string value. Usually there is just a single key/value pair, but
```

Comment:
Might help to be more explicit about encoding, e.g.

```suggestion
a string encoding of the digest using that algorithm. The named standard algorithms below use lowercase hex encoding. Usually there is just a single key/value pair, but
```

(I didn't wrap this properly)

Reply:
Done